migrations/M240409200600CreateQueueTable.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Migration\MigrationBuilder;
+use Yiisoft\Db\Migration\RevertibleMigrationInterface;
+use Yiisoft\Db\Migration\TransactionalMigrationInterface;
+
+final class M240409200600CreateQueueTable implements RevertibleMigrationInterface, TransactionalMigrationInterface
+{
+    private const TABLE_PREFIX = ''; // Add table prefix if required
+    private const QUEUE_TABLE = self::TABLE_PREFIX . 'queue';
```

Comment:
```suggestion
    private const QUEUE_TABLE = 'queue';
```

If user copy migration, then he can just change table name.